probes/hasPermissiveLicense/def.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: hasPermissiveLicense
+short: Check that the project has an permissive license.
+motivation: >
+  A permissive license allows users to use the analyzed component to be used in derivative works. Non-permissive licenses (as copyleft licenses) might be a legal risk for potential users.
+implementation: >
+ The implementation checks whether a permissive license is present
+outcome:
+  - If a license file is found and is permissive, the probe returns a single OutcomePositive.
+  - If a license file is missing the probe returns a single OutcomeNotApplicable.
+  - If the license is not permissive, the probe returns a single OutcomeNegative.
```

Comment:
Thoughts on no license file also being OutcomeNegative?

Reply:
Sounds good to me